packages/dds/tree/src/tree/pathTree.ts

```diff
@@ -32,3 +32,13 @@ export interface UpPath {
      */
      readonly parentIndex: number; // TODO: field index branded type?
 }
+
+export function getDepth(path: UpPath): number {
```

Comment:
It's not clear if a detached field is depth 0 or 1, this clarifies.

We could also define "undefined" to have depth 0 and paths in the detached sequence to have depth 1. If there is a case where that's useful we could do it (it makes the special root less special I giess?)

```suggestion
/**
 * @returns the number of nodes above this one.
 * Zero when parent is undefined and thus the path is part of a detached field.
 */
export function getDepth(path: UpPath): number {
```

Reply:
Thanks that did need clarification.

I'm confused about the "thus the path is part of a detached field" part: do you consider the document root to be part of a detached field? Is the notion of a detached field even relevant here?

Reply:
Ya, that phrasing is not super clear. I'm trying to describe paths of 0 height, which corresponding to a node that's parent node is "undefined" and parent field key is actually a detached sequence. If you read the documentation for UpPath, you can see it specially call out detached fields in the top level doc, the comment for parent, and the comment for parentField.

Reply:
I see. How about "Zero when the path's parent is undefined, meaning the path represents a node in a detached field."?

Reply:
sounds good